InvokePlaster.ps1

```diff
@@ -448,7 +448,7 @@ __________.__                   __
             $gciParams = @{}
             $parent = Split-Path $srcRelPath -Parent
             $leaf = Split-Path $srcRelPath -Leaf
-            $gciParams['LiteralPath'] = $PSCmdlet.GetUnresolvedProviderPathFromPSPath($parent)
+            $gciParams['LiteralPath'] = $PSCmdlet.GetUnresolvedProviderPathFromPSPath((Join-Path $TemplatePath $parent))
```

Comment:
Rather than these two locations, could you insert a line after 448 like:
```powershell
$srcPath = Join-Path $TemplatePath $parent
```
And then change lines 449 and 450 above to use that variable instead of $srcRelPath?  Then line 451 and 465 don't need to change.  Thanks.